vite.config.ts

```diff
@@ -6,6 +6,18 @@ import vue from '@vitejs/plugin-vue';
 
 // https://vitejs.dev/config/
 export default defineConfig( {
+	build: {
+		target: 'esnext',
```

Comment:
:+1: I really like that we can work with it now in mw code

Reply:
Can we, actually? :sweat_smile: I think I just copied that from somewhere, it would probably be smart to check if it’s correct or not. We still support some other old-ish browsers, after all (Proxies aren’t _that_ new). I’ll try to look into that later.

Reply:
oops, I suppose we can't :P

Reply:
https://github.com/wikimedia/design-codex/blob/b76f6d26389618ac266fb72f5747b157115ef78d/packages/codex/vite.config.ts#L10

Reply:
Thanks! Though after looking at the rest of wikimedia/design-codex@f9614ae76e4e974755a2e9a1a9927232033cf810, I notice they add `es2015` in some other places too, and we also already have `esnext` in our `.tsconfig` and should probably fix that. I guess that can be another pull request before this one.

Reply:
#23